### PR TITLE
Refactor paper display

### PR DESCRIPTION
**Pull request by josiahcoad**

### CODE REVIEW

File: src/PaperScreening/Components/PaperBody/PaperBody.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  changePaperDecision(decision) {
+    let paper = this.props.paper;
+    paper.decision = decision;
+    this.props.updatePaper(paper);
+    this.props.incrementRow();
+  }
+
+  render() {
+    return (
+      <PaperBodyPresentational
```
Comment (elishacoad) [inline src/PaperScreening/Components/PaperBody/PaperBody.js]: I like the splitting of the HTML and JS; makes it much easier to read.

File: src/PaperScreening/Components/Papers/PaperPanelPresentational.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/** A paper panel is the clickable banner that opens the paper body.
```
Comment (elishacoad) [inline src/PaperScreening/Components/Papers/PaperPanelPresentational.js]: I think the complete independencies of the Paper Panel code and the Paper Body code really assists readability and clarity, nice. Panels have been just working out a lot better than our previous method.

File: src/PaperScreening/Components/Papers/PapersPresentational.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { PanelGroup } from 'react-bootstrap';
+import PaperPanelPresentational from './PaperPanelPresentational';
+import React from 'react';
+
+const PapersPresentational = (props) => {
```
Comment (elishacoad) [inline src/PaperScreening/Components/Papers/PapersPresentational.js]: Said this before, but presentationals are great! (thanks @pauescalantec)

File: src/Actions/index.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
-export const changeDecision = (paper) => {
+export const updatePaper = (paper) => {
```
Comment (elishacoad) [inline src/Actions/index.js]: Good term change. Helps with forward compatibility for any new features we choose to add.

File: src/Elements/DecisionButtonGroup.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import { Colors, Decision } from './constants'
+
+import { Button } from 'react-bootstrap';
+import React from 'react'
+
+const DecisionButtonGroup = (props) =>
+    <div>
+        <Button
```
Comment (pauescalantec) [inline src/Elements/DecisionButtonGroup.js]: Liking this object but it's not really generic enough to be an element, seems pretty static and can not display anything other that exclude, include, maybe. It should receive a prop that will list or map those options to buttons, what if I want 4 or 5 buttons? Needs work, but the abstraction is a great idea.

File: src/PaperScreening/Components/PaperBody/PaperBodyPresentational.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+                    />
+                </div>
+                {props.paper.fulltextlink && (
+                    <a
+                        href={props.paper.fulltextlink}
+                        target="_blank"
+                        style={{ "fontstyle": "italic" }}
+                    >
+                        Link to Article
+                        </a>
```
Comment (pauescalantec) [inline src/PaperScreening/Components/PaperBody/PaperBodyPresentational.js]: I think this </a> closing tag has weird indentation

File: src/PaperScreening/Components/Papers/Papers.js [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+import { selectRow } from '../../../Actions';
+
+class Papers extends Component {
+    constructor(props, context) {
+        super(props, context);
+
+        this.applyGroupLogic = this.applyGroupLogic.bind(this);
+        this.matchesGroupCriteria = this.matchesGroupCriteria.bind(this);
+        this.matchesGroupsCriteria = this.matchesGroupsCriteria.bind(this);
+        this.areElibleToShow = this.areElibleToShow.bind(this);
```
Comment (pauescalantec) [inline src/PaperScreening/Components/Papers/Papers.js]: areEligibleToShow****

File: src/PaperScreening/Components/Papers/PapersPresentational.js (hunk 3 above)
Comment (pauescalantec) [reply src/PaperScreening/Components/Papers/PapersPresentational.js]: They're such a relief in the long runnnn

File: src/Elements/DecisionButtonGroup.js (hunk 5 above)
Comment (josiahcoad) [reply src/Elements/DecisionButtonGroup.js]: Ok, I left the updates in that new branch. I'll make a PR for it tooooo

**Pull request merged**